rest-client.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |s|
   s.add_dependency('http-cookie', '>= 1.0.2', '< 2.0')
   s.add_dependency('mime-types', '>= 1.16', '< 4.0')
   s.add_dependency('netrc', '~> 0.8')
+  s.add_dependency('json_pure')
```

Comment:
Ruby [ships](http://ruby-doc.org/stdlib-2.0.0/libdoc/json/rdoc/JSON.html) with `JSON`, no need for a gem. Just two lines below, Ruby 2.0.0 is a requirement, which has JSON support.